packages/web/src/js/utils/ScrollableBehavior.ts

```diff
@@ -0,0 +1,44 @@
+import { SelectorEngine } from '../dom';
+
+class ScrollableBehavior {
+  body: HTMLElement;
+  className: string;
+  selector: string;
+  offsetY: number;
+
+  constructor() {
+    this.body = SelectorEngine.findOne('body') as HTMLElement;
+    this.className = 'is-scrolling-disabled';
+    this.selector = 'dialog[open]';
```

Comment:
I would make a const from this outside of the class, e. g. `DIALOG_OPEN_SELECTOR`. The `selector` is too general.

Reply:
done